EIPS/eip-5865.md

```diff
@@ -0,0 +1,235 @@
+
+## Specification
+
+The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY” and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+NFT metadata structured by the official [EIP-721](./eip-721.md) metadata standard or the Enjin Metadata suggestions are as follows referring to OpenSea docs.
+
+```json
+{
+  "description": "Friendly OpenSea Creature that enjoys long swims in the ocean.", 
+  "external_url": "https://openseacreatures.io/3", 
+  "image": "https://storage.googleapis.com/opensea-prod.appspot.com/puffs/3.png", 
+  "name": "Dave Starbelly",
+  "attributes": {} 
+}
+```
+
+In addition to the existing NFT metadata standard, the NFT avatar standard supplements the “avatars” section. The data required for the avatar varies by metaverse platform. In the instance of Zep, image URL and animation behavior are defined to maximize the scalability of individual NFT. This standard represents a metadata extension of [EIP-721](./eip-721.md).
+
+```json
+{
+  "description": "Friendly OpenSea Creature that enjoys long swims in the ocean.", 
+  "external_url": "https://openseacreatures.io/3", 
+  "image": "https://storage.googleapis.com/opensea-prod.appspot.com/puffs/3.png", 
+  "name": "Dave Starbelly",
+  "attributes": {}, 
+  "avatars": {}
+}
+```
+
+Each property, corresponding to [EIP-721](./eip-721.md), is defined as below.
+
+| Variable | Description |
+| -------- | --------------------------------------------- |
+| name | The name of the item. |
+| description | A description of the item. |
+| external_url | An external URL that will appear below the asset's image. |
+| image | An URL to the image of the item. This can be any type of image, including SVG, PNG, IPFS URL, or paths. The recommended size of an image is a width between 320 and 1080 pixels and an aspect ratio between 1.91:1 and 4:5 inclusive. |
+| attributes | An attribute for the item, which will show up on the NFT marketplace. |
+| avatars | A metaverse platform where the item can be integrated as NFT avatars. |
+
+Under "avatars", metadata keys for any metaverse platform, such as Zep.us and Another.world, can be added. The value included in the key will define the animation of a character.
+
+```json
+{
+   "avatars": {
+         "zep": {},
+         "another.world": {}
+   }
+}
+```
+
+This proposal currently only includes the standard of Zep NFT avatars, but the standard of additional services can be also defined under "avatars".
+
+```json
+{
+  "zep": {  
+             "image": "IMAGE URL",
+             "frame_width": 48,
+             "frame_height": 64,
+             "animations": {}
+  }
+}
+```
+
+Each property is defined as below.
```

Comment:
Are these properties given as part of a Specification, or as an example? Are implementations required or recommended to use the variables other than what's already in the following tables?